functions/system.sh

```diff
@@ -276,7 +276,7 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
 
   # Find current settings
   if is_pi && grep -q "enable_uart=1" /boot/config.txt; then sel_1="ON"; else sel_1="OFF"; fi
-  if is_pithree && grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then sel_2="ON"; else sel_2="OFF"; fi
+  if ( is_pithree || is_pithreeplus ) && grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then sel_2="ON"; else sel_2="OFF"; fi
```

Comment:
why braces here?

Reply:
Ah I see. In bash this is in fact not needed.

https://unix.stackexchange.com/a/88851

Reply:
i will change this in the new PR